### ISSUE TITLE
For Search, Have a Selection to Include or Exclude Resuts from Accounts with the "Bot' Label 

### ISSUE THREAD

**Issue by Timothyjchambers** [mastodon/mastodon#24588]
### Pitch

When doing a search for Users, and potentially for ALL searches, have a UI element that lets users choose to include or exclude results from accounts that are labeled "Bot."

https://docs.joinmastodon.org/user/profile/#bot

Then the results of any search string can include or exclude all such accounts, or content from all such accounts.

### Motivation

Currently - again especially in the use case of searching for users, many "bot" accounts including many well-meaning accounts that crosspost from Twitter - return in the results. This can be vexing to users looking for authentic, non-scripted accounts. 

Real-world example of a frustrating search for this user: 
https://mastodon.world/@wigbert/110220854950202272

So when users are looking for non-bot accounts, having this functionality will make for much more clean, helpful and accurate results.

**Comment by nekromoff** [comment]
I agree, finding timeline spammed by bots is annoying at least, unusable at the worst.
